maps/away/ships/freebooter/freebooter_ship.dm

```diff
@@ -5,7 +5,7 @@
 	prefix = "ships/freebooter/"
 	suffixes = list("freebooter_ship.dmm")
 
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, SECTOR_SRANDMARR, ALL_COALITION_SECTORS)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_NEW_ANKARA, SECTOR_BADLANDS, SECTOR_AEMAQ, SECTOR_SRANDMARR, ALL_COALITION_SECTORS, SECTOR_UUEOAESA)
```

Comment:
this should use the `ALL_...` lists instead of adding new sectors to it manually

and anyways the freebooter should just be `ALL_POSSIBLE_SECTORS`